src/Sylius/Behat/Context/Setup/CurrencyContext.php

```diff
@@ -173,16 +173,6 @@ public function thatChannelAllowsToShopUsingCurrency(ChannelInterface $channel,
     }
 
     /**
-     * @Given /^the exchange rate for (currency "[^"]+") was changed to ([0-9\.]+)$/
-     * @Given /^the ("[^"]+" currency) has an exchange rate of ([0-9\.]+)$/
-     */
-    public function theExchangeRateForWasChangedTo(CurrencyInterface $currency, $exchangeRate)
-    {
-        $currency->setExchangeRate($exchangeRate);
-        $this->saveCurrency($currency);
-    }
```

Comment:
Why removing this? In this PR you should only remove exchange rate from order. Removing it from currency will break a lot of things and should be done in separate PR.